job-dsls/jobs/pr_jobs.groovy

```diff
@@ -95,7 +95,7 @@ def final REPO_CONFIGS = [
         ],
         "jbpm-work-items"           : [
                 label      : "kie-linux && kie-mem4g",
-                timeoutMins: 60,
+                timeoutMins: 120,
```

Comment:
i think the previous rans proved the time is 90 mins or so, so 120 timeout is ok